docs/index.rst

```diff
@@ -11,3 +11,7 @@ Contents:
    hdfs_connector
    configuration_options
    changelog
+
+The HDFS connector is available as open source software under the
+`Apache License v2.0 license <http://www.apache.org/licenses/LICENSE-2.0>`_.
+Source code at `https://github.com/confluentinc/kafka-connect-hdfs <https://github.com/confluentinc/kafka-connect-hdfs>`_.
```

Comment:
nit: Do we still need the full stop in the end? Click is fine, but copy pasting might be more difficult

Reply:
by "full stop" do you mean the period at the end of the sentence? yes i think so, since a sentence is incomplete without a period to end it. i would assume most users would simply click the hyperlink, not copy/paste anyway.

Reply:
I'm fine with that. Just wanted to ask.